lib/pengine/clone.c

```diff
@@ -412,51 +412,57 @@ clone_print(resource_t * rsc, const char *pre_text, long options, void *print_da
     for (; gIter != NULL; gIter = gIter->next) {
         gboolean print_full = FALSE;
         resource_t *child_rsc = (resource_t *) gIter->data;
+        gboolean partially_active = child_rsc->fns->active(child_rsc, FALSE);
 
         if (options & pe_print_clone_details) {
             print_full = TRUE;
         }
 
-        if (child_rsc->fns->active(child_rsc, FALSE) == FALSE) {
-            /* Inactive clone */
-            if (is_set(child_rsc->flags, pe_rsc_orphan)) {
-                continue;
-
-            } else if (is_set(rsc->flags, pe_rsc_unique)) {
+        if (is_set(rsc->flags, pe_rsc_unique)) {
+            // Print individual instance when unique (except stopped orphans)
+            if (partially_active || is_not_set(rsc->flags, pe_rsc_orphan)) {
                 print_full = TRUE;
+            }
+
+        // Everything else in this block is for anonymous clones
+
+        } else if (is_set(options, pe_print_pending)
+                   && (child_rsc->pending_task != NULL)
+                   && strcmp(child_rsc->pending_task, "probe")) {
```

Comment:
regarding 2 last conditions, found at least 5 other occurrences like that,
perhaps appealing to wrap that into dedicated inline function, say
`crm_str_1st_undef_or_eq`, allowing for
`!crm_str_1st_undef_or_eq(child_rsc->pending_task, "probe")`
compaction.